pkgs/applications/science/misc/snakemake/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchurl, python3Packages }:
+
+let
+  name = "snakemake";
+  version = "5.2.2";
+in python3Packages.buildPythonApplication {
+  name = "${name}-${version}";
+
+  src = fetchurl {
```

Comment:
Any reason not to use `fetchFromBitBucket` here?

Reply:
Nope, only a noob not knowing better. :-)